src/qtgui/dockfft.cpp

```diff
@@ -41,6 +41,11 @@
 #define DEFAULT_FFT_AVG         25
 #define DEFAULT_COLORMAP        "gqrx"
 
+static const std::vector<const char*>window_strs({
+    "hamming", "hann", "blackman", "rectangular", "kaiser",
+    "blackmanharris", "bartlett", "flattop"
+});
```

Comment:
This could be simplified:
```suggestion
static const QStringList window_strs = {
    "hamming", "hann", "blackman", "rectangular", "kaiser",
    "blackmanharris", "bartlett", "flattop"
};
```